x/concentrated-liquidity/types/genesis/genesis.go

```diff
@@ -7,7 +7,7 @@ import (
 // DefaultGenesis returns the default GenesisState for the concentrated-liquidity module.
 func DefaultGenesis() *GenesisState {
 	return &GenesisState{
-		PoolData:       []*PoolData{},
```

Comment:
Out of curiosity, what's the reason we deference this?

Reply:
We want to refrain from using pointers if possible for clarity and better readability and safetyness